01-lets_get_started/01.rb

```diff
@@ -0,0 +1,9 @@
+print'enter side of metrix'
+hash = gets.to_i
+print '#' * hash + "\n"
+sl = hash - 2
+val = '#' + ' ' * sl + '#'
+sl.times do
+  puts val
+end
+print '#' * hash
```

Comment:
This is a duplication of a previous line; perhaps try to make it a method & call it instead.

Note: The code is actually very good, I am actually very happy to see it. Its very simple & intuitive. And also without using any conditions! Thats a very nice attribute.